api/envoy/config/core/v3/udp_socket_config.proto

```diff
@@ -21,4 +21,10 @@ message UdpSocketConfig {
   // defaults to 1500 bytes.
   google.protobuf.UInt64Value max_rx_datagram_size = 1
       [(validate.rules).uint64 = {lt: 65536 gt: 0}];
+
+  // Configures whether `Generic Receive Offload (GRO)
+  // <https://en.wikipedia.org/wiki/Large_receive_offload>`_ is enabled (if the operating system
+  // supports it) when reading from the UDP socket. The default is context dependent and is
+  // documented where UdpSocketConfig is used.
+  google.protobuf.BoolValue use_gro = 2;
```

Comment:
I thought GRO was still compiled as an extension. 
Do we need to do the compile options here or do checks somewhere that the GRO code is included as part of config validation?

Reply:
No GRO is always in, GSO is the extension right now which we (you :)) need to fix.

Reply:
Right, of course just testing you, I would never get my UDP acronyms confused ;-)
(haha, thanks)